.gitignore

```diff
@@ -172,5 +173,7 @@ qt/res/Info.plist
 
 designer_version.h
 
+.gradle/
```

Comment:
Why do you need this? We already have it in android/.gitignore

Reply:
fixed